src/test/java/lotto/model/BunchOfLottoTest.java

```diff
@@ -1,23 +1,44 @@
 package lotto.model;
 
-import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
 
 import java.util.Set;
 
-import static lotto.util.TypeConvert.convertStringToLottoNumberSet;
+import static lotto.util.TypeConverter.convertStringToLottoNumberSet;
+import static org.assertj.core.api.Assertions.assertThat;
 
 public class BunchOfLottoTest {
+    @DisplayName("수동으로 생성된 로또 추가")
+    @ParameterizedTest
+    @ValueSource(strings = {"1,2,3,4,5,6"})
+    void addLottoTest(String inputNumber) {
+        BunchOfLotto bunchOfLotto = new BunchOfLotto();
+        bunchOfLotto.addLotto(LottoGenerator.makeManualLotto(inputNumber));
+
+        assertThat(bunchOfLotto.getBunchOfLotto().size()).isEqualTo(1);
+    }
+
+    @DisplayName("자동으로 생성된 로또들 추가")
+    @Test
+    void addBunchOfLottoTest() {
+        BunchOfLotto bunchOfLotto = new BunchOfLotto();
+        bunchOfLotto.addBunchOfLotto(LottoGenerator.makeBunchOfAutoLotto(5));
+
+        assertThat(bunchOfLotto.getBunchOfLotto().size()).isEqualTo(5);
```

Comment:
```suggestion
        assertThat(bunchOfLotto.getBunchOfLotto()).hasSize(5);
```
이렇게 변경해볼 수 있을 것 같아요 😃 (위의 테스트도 동일하게 바꿔볼 수 있을 것 같아요 😃)